app/views/GovukWrapper.scala.html

```diff
@@ -49,8 +49,8 @@
   feedbackOnSignOut: Boolean = true)(implicit request: Request[_], messages: Messages)
 
 @linksElement = {
-<link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/palette.css")'>
-<link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/focus.css")'>
+<link rel="stylesheet" type="Text/css" href='@routes.Assets.at("stylesheets/palette.css")'>
```

Comment:
You accidentally changed the capitalisation of `text` to `Text` in a few files, you'll need to go through and check for any more